### PR TITLE
[SWA-146][FEAT] - Add Openocean support

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-146](https://linear.app/swaprhq/issue/SWA-146/add-openocean-support)

# Description
* Add Kleros as a routing platform

# Visual Evidence
image

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Connect your wallet
5) Go to the Swapbox, select the pair you want to swap on any of the supported chains by Openocean (Arbitrum, BSC, Gnosis, Ethereum, Optimism, Polygon, Scroll) and Openocean should appear as a valid route.

### CODE REVIEW

File: src/lib/eco-router/api.ts
Comment (berteotti) [inline src/lib/eco-router/api.ts:318]: This doesn't look right 😄

Comment (ElRodrigote) [reply src/lib/eco-router/api.ts:318]: Ahhh it was a test and you passed (?
Lol sorry, addressing it right now

**Review by Diogomartf — APPROVED** [review]

**Review by berteotti — APPROVED** [review]
Good job, looking good!

**Comment by berteotti** [comment]
Don't forget to merge this only after updating Swapr SDK version

**Review by berteotti — APPROVED** [review]
Looking good, don't forget to rebase and squash commit before merging

**Pull request merged**
